Dockerfile

```diff
@@ -0,0 +1,25 @@
+# 1. Builder image
+FROM node:18-alpine AS compile-image
+
+ # install git and get bare github data for GitRevisionPlugin
+RUN apk update; \
+    apk add --no-cache git; \
+    git clone --bare --depth=1 https://github.com/secvisogram/secvisogram.git /work/
```

Comment:
Should we really clone here or better use the file from the checked out repo?

Reply:
It is only a bare clone, we only need the information for GitRevisionPlugin.